wca-regulations.md

```diff
@@ -26,6 +26,9 @@ WCA Regulations in [PDF format](link:pdf)
 ### Source
 Development of the WCA Regulations and Guidelines is public [on GitHub](https://github.com/thewca/wca-regulations) and the discussion is public [on the WCA Forum](https://forum.worldcubeassociation.org/c/regulations).
 
+### License
+This document is licensed under a [CC BY-NC-ND 4.0](https://creativecommons.org/licenses/by-nc-nd/4.0/) license. Permissions beyond the scope of this license may be available at [thewca/wca-regulations-translations](https://github.com/thewca/wca-regulations-translations#license)
```

Comment:
Since this text appears on the main Regulations, I think we should avoid linking to something else that can imply the availability of a looser license.

```suggestion
This document is licensed under a [CC BY-NC-ND 4.0](https://creativecommons.org/licenses/by-nc-nd/4.0/) license.
```

If we want translations to specify looser languages, we could generalize the mechanism we use for "this translation is not an official version" at https://github.com/thewca/wca-regulations-translations#translation-instructions

Reply:
is this a change for this regulations cycle iyo? @lgarron

Reply:
This is the common procedure that Creative Commons provides to grant more rights (in this case, the right to distribute a translation). See "Simplest CC+ Example" [here](https://wiki.creativecommons.org/wiki/CCPlus).

However, we may simply use the CC BY-NC-ND 4.0 and only allow translations to be posted in our website.